standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/common/histogram/KllHistogramEstimator.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hive.common.histogram;
+
+import org.apache.datasketches.kll.KllFloatsSketch;
+import org.apache.hadoop.hive.common.histogram.kll.KllUtils;
+import org.apache.hadoop.hive.common.type.HiveDecimal;
+import org.apache.hadoop.hive.ql.util.JavaDataModel;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class KllHistogramEstimator {
```

Comment:
remove these classes from the metastore...

Reply:
should we keep those classes in hive-exec module?

Reply:
maybe `org.apache.hadoop.hive.ql.udf.generic.histogram` in hive-exec?  @szlta what do you think?

Reply:
I like your suggestion, the only concern I have is that `KllHistogramEstimator` is the histogram's equivalent of [`NumDistinctValueEstimator`](https://github.com/apache/hive/blob/8b48ce50684791d53dcb942958de233263cc421f/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/common/ndv/NumDistinctValueEstimator.java) for NDV, I thought it might be confusing to place it elsewhere, but I am fine with whatever you and @szlta suggest.

Reply:
`org.apache.hadoop.hive.ql.udf.generic.histogram` sounds good to me

Reply:
I have moved the classes to `org.apache.hadoop.hive.ql.udf.generic.histogram` as suggested, it looks indeed better, thanks!